articles/other/other-how-delete-sso-user.md

```diff
@@ -0,0 +1,190 @@
+---
+title: How to delete single sign on users | UKCloud Ltd
```

Comment:
Perhaps 'delete' could be better worded as 'remove access from' – IDAM users don't have the necessary access to be able to delete a user from the system entirely, but they can remove the user's access to services and accounts through the process outlined in the article.

Reply:
Fixed

Reply:
The latter half of the document still refers to this as deleting an SSO user. Also, it's dawned on  me that 'removing access **rights**' sounds a bit better now.